docs/userguide/scheduling/resource-propagating.md

```diff
@@ -272,6 +272,21 @@ spec:
     ...
 ```
 
+## PropagationPolicy/ClusterPropagationPolicy priority
```

Comment:
A little rough. For a user who does not read the code, may have a series of questions:
1. What is "implicit priority"? By default it is?
2. How do I config explicit priority?
3. Typical use scenario?
4. I suggest adding a specific example.

Reply:
I add specific example, please take a look